addons/mail/static/src/core/thread_service.js

```diff
@@ -900,8 +900,8 @@ export class ThreadService {
         const partner_ids = validMentions?.partners.map((partner) => partner.id);
         if (!isNote) {
             const recipientIds = thread.suggestedRecipients
-                .filter((recipient) => recipient.persona && recipient.checked)
-                .map((recipient) => recipient.persona.id);
+                .filter((recipient) => recipient.checked)
+                .map((recipient) => recipient.persona?.id || recipient.email);
```

Comment:
I dont think that `recipientIds` should contain emails (or the name is now misleading), it looks like it is used as an param for `/mail/message/post` and will crash at https://github.com/odoo/odoo/blob/saas-16.3/addons/mail/models/mail_thread.py#L2012-L2017